guides/testing/index.md

```diff
@@ -1,78 +1,78 @@
+Tout d'abord, vous pouvez exécuter la suite de tests avec la commande `ember test`, ou `ember t`, dans votre terminal. Ceci exécutera la suite de test une seule fois.
 
-Suppose, instead, you want the suite to run after every file change. You can enter `ember test --server`, or `ember t -s`.
+Si vous souhaitez que la suite s'exécute après chaque modification de fichier. Vous pouvez saisir  `ember test --server`, or `ember t -s`.
 
-Lastly, if you are already running a local development server (through `ember server`), you can visit the `/tests` URI. This will render the `tests/index.html` template.
+enfin, si vous exécutez déjà un serveur de développement local (via `ember server`), rendez-vous sur l'URI `/tests`. Vous y trouverez le modèle `tests/index.html`.
 
 ```bash
-# Run all tests once
+# Exécute tous les tests une fois
 ember test
 ember t
 
-# Run all tests after every file change
+# Exécute tous les tests après chaque modification de fichier.
 ember test --server
 ember t -s
 ```
 
-### How to Filter Tests
+### Comment filter les tests
 
-When you are working on a single component or page, you will want only a small subset of tests to run after every file change. To specify which tests to run, you can add `--module` or `--filter` option to your command.
+Lorsque vous travaillez sur un seul composant ou une seule page, vous ne voulez exécuter qu'un petit sous-ensemble de tests après chaque modification de fichier. Pour spécifier les tests à exécuter, vous pouvez ajouter l'option `--module` ou `--filter` à votre commande.
 
-The `--module` option allows you to select a **module**—a group of tests that you specified in `module()` in QUnit.
+L'option `--module` vous permet de sélectionner un **module**— un groupe de tests que vous avez spécifiés dans `module()` dans QUnit.
 
 ```bash
-# Button component example
+# Exemple de composant Button
 ember test --server --module="Integration | Component | simple-button"
 
-# Run tests for a location service
+# Exécute des tests pour un service de localisation# Run tests for a location service
 ember t -s -m="Unit | Service | location"
 ```
 
-The `--filter` option is more versatile. You can provide a phrase to match against the modules and test descriptions. A test description is what appears in `test()` in QUnit.
+L'option `--filter` est plus polyvalente. Vous pouvez fournir une phrase à comparer aux modules et aux descriptions des tests. Une description de test est ce qui apparaît dans `test()` dans QUnit.
 
 ```bash
-# Button component example
-ember test --server --filter="should show icon and label"
+# Exemple de composant Button
+ember test --server --filter="devrait afficher l'icône et l'étiquette"
 
-# Test everything related to your dashboard
-ember t -s -f="Dashboard"
+# Tester tout ce qui est lié à votre tableau de bord
+ember t -s -f="Tableau de bord"
 
-# Run integration tests
-ember t -s -f="Integration"
+# Exécuter des tests d'intégration
+ember t -s -f="Intégration"
```

Comment:
Here, we are talking about the "tests/integration" folder, so the filter is not translated.

```suggestion
ember t -s -f="Integration"
```